lib/author.rb

```diff
@@ -0,0 +1,33 @@
+class Author
+  attr_reader :name,
+              :books
+  def initialize(data)
+    @first_name = data[:first_name]
+    @last_name = data[:last_name]
+    @name = "#{@first_name} #{@last_name}"
```

Comment:
If we're calculating this dynamically based off existing instance variables, we can make this it's own method. The less instance variables the better.